packages/koa-metrics/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/shopify/quilt/blob/master/packages/koa-metrics/README.md",
   "dependencies": {
+    "@shopify/statsd": "^1.0.0",
```

Comment:
can I do this yet? I remember us discussing a fix recently. 
Or would I need to wait till `@shopify/statsd` is actually publish.

Reply:
ya I can't...grr...